cpu/cc26xx-cc13xx/dev/gpio-interrupt.c

```diff
@@ -95,7 +95,10 @@ gpio_interrupt_isr(void)
   for(i = 0; i < NUM_IO_MAX; i++) {
     /* Call the handler if there is one registered for this event */
     if((pin_mask & (1 << i)) && handlers[i] != NULL) {
-      handlers[i](i);
+      enabled = HWREG(IOC_BASE + IOC_O_IOCFG0 + (i * 4)) & IOC_IOCFG0_EDGE_IRQ_EN;
+      if (enabled) {
```

Comment:
You should remove the space before the opening parenthesis here.

Why use a new variable instead of putting the entire check inside the if condition?